kolibri/core/assets/src/exams/utils.js

```diff
@@ -96,8 +96,8 @@ export function convertV2toV3(questionSources, exam) {
   const questions = annotateQuestionsWithItem(questionSources);
   return {
     section_id: uuidv4(),
-    section_title: '',
-    description: '',
+    section_title: 'Section one',
+    description: 'Sample description',
```

Comment:
One thing to note is that we should probably revert this. It is causing the tests to fail and if we are going to introduce a title and/or description by default we'll need to use a translated string and account for it in the tests. We can handle this in a separate follow-up issue

Reply:
Oops! I think I missed it before committing.